### ISSUE TITLE
Compare sets of common traits

### ISSUE THREAD

**Issue by jamesdabbs** [pi-base/viewer#24]
Given a collection of spaces (say, with a known property), it would be nice to compare the traits that those spaces have in common.

**Comment by StevenClontz** [comment]
Suggested by David Milovich

**Comment by jamesdabbs** [comment]
@StevenClontz - do you want to take a swing at mocking out a UI for this? 

My big questions are:

* How do you specify which spaces you are interested in? 
* What do you want to see about commonalities? Each of the properties that all of the spaces in that set have in common?

**Comment by StevenClontz** [comment]
Yes, I'll work on this when I'm back in the office in two weeks.

**Comment by StevenClontz** [comment]
Actually I was able to sneak into the office for a couple hours today. I've got the simplest thing mocked up: choose up to 4 spaces, and simply display the properties they have in common.

I'm sure there are more interesting/elegant insights we could tease out of the data than this, but it's a start.
